.github/workflows/docs-production-netlify-cli.yml

```diff
@@ -0,0 +1,144 @@
+name: Docs Production Deployment - Netlify CLI
+
+env:
+  NODE_VERSION: 20
+  DOCSEARCH_API_KEY: ${{ secrets.DOCSEARCH_API_KEY }}
+  DOCSEARCH_APP_ID: ${{ secrets.DOCSEARCH_APP_ID }}
+
+on:
+  create:
+    branches:
+      - 'prod-docs/[0-9]+\\.[0-9]+'
+  push:
+    branches:
+      - 'prod-docs/[0-9]+\\.[0-9]+'
+      - feature/dev-issue-1790
+  workflow_dispatch:
+
+defaults:
+  run:
+    working-directory: ./docs/
+
+jobs:
+  check-version:
+    name: Check Docs Version and Branch
+    runs-on: ubuntu-latest
+    outputs:
+      continue: ${{ steps.check-branch.outputs.continue }}
+      docs_latest_version: ${{ steps.set-docs-version.outputs.docs_latest_version }}
+
+    steps:
+      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # https://github.com/actions/checkout/releases/tag/v4.1.1
+
+      - name: Obtain the docs latest version
+        working-directory: ./
+        id: set-docs-version
+        run: |
+          docs_latest_version=$(git ls-remote --heads origin | grep 'prod-docs/[0-9]\+\.[0-9]\+' | sed 's|.*/prod-docs/||' | sort -V | tail -n 1)
+          echo "The docs latest version: $docs_latest_version"
+          echo "DOCS_LATEST_VERSION=$docs_latest_version" >> $GITHUB_ENV
+          echo "docs_latest_version=$docs_latest_version" >> $GITHUB_OUTPUT
+
+      - name: Validate Docs version
+        run: |
+          if [ -z "$DOCS_LATEST_VERSION" ]; then
+            echo "Docs version is empty";
+            exit 1;
+          fi
+
+      - name: Check if branch matches the latest version
+        id: check-branch
+        run: |
+          FEATURE_BRANCH="feature/dev-issue-1790"
```

Comment:
Remember to remove the feature branch from the code before merging the PR.

Reply:
Feature removed